pymeasure/instruments/siglenttechnologies/siglent_spdbase.py

```diff
@@ -98,92 +98,64 @@ def __init__(self, adapter, **kwargs):
         get_process=lambda v: SystemStatusCode(int(v, base=16)),
     )
 
-    local_lockout = Instrument.setting(
+    disable_local_interface = Instrument.setting(
```

Comment:
A verb in the name implies an "action", which is most properly a method, not a property, see https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#your-instrument-s-user-interface.